packages/components/src/Menu/Menu.module.css

```diff
@@ -94,6 +101,11 @@
   color: var(--color-destructive);
 }
 
+.triggerWrapper {
+  display: inline-flex;
+  outline: none;
```

Comment:
I'm unsure if this is the right thing to do or not.

if you provide an interactive component like a Button or a Chip, then the outline and focus states are present

if you provide simple text, or say an Icon - those aren't interactive in their natural state and don't ship with an outline.

so in those cases, you actually do want to have this for the trigger otherwise you have no indication that your trigger element is focused

Reply:
this problem also stems from the `<div role="button">`

once we solve that properly, it gets much simpler.

thinking on it more, it might actually be fine. you shouldn't provide only Text or an Icon anyways. you should be wrapping that in a Button, or some sort of interactive role. so yeah once we solve that wrapper issue, this goes away and we document that your trigger MUST be an interactive element, and handle focus styles.